inst/pages/30_differential_abundance.qmd

```diff
@@ -331,89 +331,6 @@ This will create a folder that is called like in the output specified
 above. It contains also figures to visualize difference between
 significant taxa.
 
-### LinDA 
-
-Lastly, we cover linear models for differential abundance analysis of
-microbiome compositional data (@Zhou2022). This is very similar to
-ANCOMBC with few differences: 1) LinDA corrects for the compositional
-bias differently using the mode of all regression coefficients. 2) it
-is faster (100x-1000x than ANCOMBC and according to the authors); 3)
-it supports hierarchical models. The latest ANCOMBC versions are also
-supporting hierarchical models. Nevertheless, LinDA seems a promising
-tool that achieves a very good power/fdr trade-off together with
-ANCOMBC according to the review. The speed improvements might make it
-critical especially for datasets that have higher sample or feature
-set sizes.
```

Comment:
You could keep this explanation and move it to the end of the section under subtitle: "Further notes on differential abundance". Modify a bit to make the text fit the fact that the examples were removed.

Reply:
Sure, noted.